### PR TITLE
format: add missing #include <stdint.h>

### PR DESCRIPTION
**Pull request by tlauda**
Adds missing stdint header, since we are using types
defined in that file. Right now format.h compiles
by accident.

Signed-off-by: Tomasz Lauda <tomasz.lauda@linux.intel.com>

### CODE REVIEW

File: src/include/sof/audio/format.h [hunk 1]
```diff
@@ -10,6 +10,8 @@
 #ifndef AUDIO_FORMAT_H
 #define AUDIO_FORMAT_H
```
Comment (dbaluta) [inline src/include/sof/audio/format.h]: @tlauda Please explain in the commit message why is this change needed. I assume that this include is needed to fix some compilation error, right?

Comment (tlauda) [reply src/include/sof/audio/format.h]: @dbaluta Done.

**Comment by dbaluta** [comment]
@tlauda all looks good now, will merge the changes later after CI finishes all the checks.

**Comment by michalgrodzicki** [comment]
@xiulipan, could you provide more details about jenkins CI issue?

**Comment by michalgrodzicki** [comment]
@xiulipan  could you provide more details about jenkins CI issue?

**Comment by xiulipan** [comment]
@michalgrodzicki @tlauda 
Please see my announcement in the mail list.
